src/olympia/addons/models.py

```diff
@@ -1271,18 +1271,24 @@ def is_unreviewed(self):
     def is_public(self):
         return self.status == amo.STATUS_PUBLIC and not self.disabled_by_user
 
-    def is_incomplete(self):
-        # Don't hassle add-ons with only unlisted versions.
-        if not self.is_listed:
-            return False
-
+    def has_complete_metadata(self, for_channel=None):
```

Comment:
You're never using the `for_channel` argument in this method or the one below. Remove it ?